.github/workflows/test.yaml

```diff
@@ -1,8 +1,19 @@
-name: Python Test Suite
+name: CSL CI Test Suite
 on:
 - push
 - pull_request
 jobs:
+    name: prettier
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 
+      uses: actions/checkout@v2
+    - name: Set up npm
+      uses: actions/setup-node@v1
+    - name: Install prettier
+      run: npx prettier@2.0.5 . --write
+    - name: check json and yaml formatting
+      run: npx prettier --check schemas/**/*.{json,yaml}
```

Comment:
@dhimmel - github is complaining about a yaml syntax error here, but I've tried everything I can think of, and nothing works. Any ideas?

Reply:
see https://github.com/citation-style-language/schema/pull/264/files#r442869965

Reply:
OMG; what am embarrassing oversight!

Thanks.